lib/spack/docs/containers.rst

```diff
@@ -151,16 +169,17 @@ of environments:
        format: docker
 
        # Select from a valid list of images
-       base:
-         image: "centos:7"
+       images:
+         os: "centos:7"
          spack: develop
 
        # Whether or not to strip binaries
        strip: true
 
        # Additional system packages that are needed at runtime
        os_packages:
-       - libgomp
+         final:
+         - libgomp
 
        # Extra instructions
        extra_instructions:
```

Comment:
@alalazo When just following along and copying from this example I ran into the following:

```
$ spack containerize > Dockerfile
...
ruamel.yaml.scanner.ScannerError: while scanning a simple key
  in "/Users/pbryant/Development/10-Containers/spack-containerize/gromacs/spack.yaml", line 27, column 1
could not find expected ':'
  in "/Users/pbryant/Development/10-Containers/spack-containerize/gromacs/spack.yaml", line 28, column 1
```

I was edit able to edit the file in order to get it working but wanted to bring it up.

Reply:
@paulbry Sorry, I don't understand the edit. Is there an issue with any example in this PR?

Reply:
@alalazo  It's my fault, I commented on the wrong lines of code and should have corrected it. The section that caused an issue was:

```
       extra_instructions:
         final: |
   RUN echo 'export PS1="\[$(tput bold)\]\[$(tput setaf 1)\][gromacs]\[$(tput setaf 2)\]\u\[$(tput sgr0)\]:\w $ \[$(tput sgr0)\]"' >> ~/.bashrc
```

Simply copying/pasting that section is what caused the error I provided above. I just tabbed the `RUN echo ...` line over to correct this.

Reply:
Should be fixed now, thanks